src/components/Categories.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import '../styles/Card.css';
+import { useLocation } from 'react-router-dom';
+import getCategories from '../services/categoriesAPI';
+
+function Categories() {
+  const { pathname } = useLocation();
+  const category = pathname.includes('comidas') ? 'Meal' : 'Drink';
+  const [categories, setCategories] = useState();
+  getCategories(category)
+    .then((response) => setCategories(response));
+  const CINCO = 5;
+  return categories ? (
+    <>
+      { categories
+        .filter((_, index) => index < CINCO)
+        .map((element, index) => (
+          <button
+            type="button"
+            key={ index }
+          >
+            {element}
+          </button>
```

Comment:
Acho que entendi pq tá dando o erro que vc falou.
Em cada posição do array, a API retorna um objeto como o abaixo:

```javascript
{
    strCategory: "Beef"
}
```

Do jeito que vc tá fazendo, tá mandando colocar esse objeto como children do button. Acredito que a alteração sugerida abaixo seja suficiente pra corrigir o bug ^^
Também vou colocar uma sugestão pra usar como key.

```suggestion
        .map(({ strCategory }, index) => (
          <button
            type="button"
            key={ `strCategory${index}` }
          >
            { strCategory }
          </button>
```